mne/preprocessing/interpolate.py

```diff
@@ -72,7 +73,7 @@ def equalize_bads(insts, interp_thresh=1., copy=True):
     return insts
 
 
-def interpolate_bridged_electrodes(inst, bridged_idx):
+def interpolate_bridged_electrodes(inst, bridged_idx, bad_limit: int = 4):
```

Comment:
We don't have any other type annotations, so let's not start here. Let's discuss separately if we should do this. (I thought we did already somewhere, but with a quick issue search I didn't find anything.)